intermittent_tracker/templates/most_flaky.html

```diff
@@ -9,38 +9,42 @@
 </head>
 
 <body>
-    <form action="/most-flaky" method="get">
-        <label for="limit">Limit:</label>
-        <input type="number" id="limit" name="limit" min="0">
-        <button type="submit">Apply</button>
-    </form>
+    <div class="main-content">
+        <h2>Flakiest Tests</h2>
+        
+        <form action="/most-flaky" method="get">
+            <label for="limit">Limit:</label>
+            <input type="number" id="limit" name="limit" min="0">
+            <button type="submit">Apply</button>
+        </form>
 
-    <table class="big" border="1">
-        <thead>
-            <tr>
-                <th>#</th>
-                <th>Test Path</th>
-                <th>Number of Unexpected Results</th>
-                <th>GitHub Issue</th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for test in tests %}
-            <tr>
-                <td>{{ loop.index }}</td>
-                <td>{{ test.path }}</td>
-                <td>{{ test.total_unexpected_count }}</td>
-                <td>
-                    {% if test.issues %}
-                        <a href="https://github.com/servo/servo/issues/{{ test.issues.issues[0].number }}" rel=”noopener" target="_blank">{{ test.issues.issues[0].title }}</a>
-                    {% else %}
-                        No issues found
-                    {% endif %}
-                </td>
-            </tr>
-            {% endfor %}
-        </tbody>
-    </table>
+        <table class="big" border="1">
+            <thead>
+                <tr>
+                    <th>#</th>
+                    <th>Test Path</th>
+                    <th>Number of Unexpected Results</th>
+                    <th>GitHub Link</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for test in tests %}
+                <tr>
+                    <td>{{ loop.index }}</td>
+                    <td>{{ test.path }}</td>
+                    <td>{{ test.total_unexpected_count }}</td>
+                    <td>
+                        {% if test.issues %}
+                            <a href="https://github.com/servo/servo/issues/{{ test.issues.issues[0].number }}" rel="noopener" target="_blank">issue #{{test.issues.issues[0].number}}</a>
+                        {% else %}
+                            No issues found
+                        {% endif %}
+                    </td>
```

Comment:
Sorry If I wasn't clear before, but I was imagining a view like this:

| Test | Count |
|------- |-----------|
| `/css/path/to/flaky/test.html` (#17, #1) | 20272 |
| `/css/path/to/another/flaky/test.html` (#13, #2) | 2011 |